.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
If you wanted, it looks like GitHub's red error highlighting could be fixed with a [linguist override on this file](https://github.com/github/linguist#overrides) (suggested [on SO here](https://stackoverflow.com/a/48634215)). Not necessary since it doesn't impact actual usage!

Reply:
Will make a PR to linguist so it can start figuring this out on its own in the future